lib/builder/context.ex

```diff
@@ -0,0 +1,17 @@
+defmodule Burrito.Builder.Context do
+  use TypedStruct
+
+  alias Burrito.Builder.Target
+
+  @type erts_location :: nil | {:release | :local | :url | :unpacked, term()}
+
+  typedstruct do
```

Comment:
Nitpick: since you're enforcing all of the fields that don't have defaults, you can do this:

```elixir
typedstruct enforce: true do
  ...
end
```